src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/" method="get">
+      <div class="input-wrapper">
```

Comment:
Why can't you assign this class to the form tag? And instead you create another additional div tag ?